wordpress-mobile/WordPress-Android
# Move me to bottom nav

Moves Me (avatar) to Bottom Navigation as a fourth tab after Notifications on JP app, and in menu under manage on WP app
- Updates MySite Info layout
- Updates Me screen layout

Fixes # pcdRpT-2HW-p2#comment-4875, and this discussion p1686812471360099-slack-C04SFL0RP51

**Jetpack app**
| Me at the Bottom | Me Screen | Dark Mode
|--------|--------|--------|
|![Screenshot_20230618_160803](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/d4296796-00f6-4e29-95de-ee74962c147f)|![Screenshot_20230618_160742](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/33cb9041-7f93-45df-ad1c-e971a66d526e)|![Screenshot_20230618_162015](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/95f20821-e048-449a-9164-e239781ee085)|

**WordPress app**
| Me in menu | Me |
|--------|--------|
|![Screenshot_20230620_171528](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/8d5c6c20-bc31-47df-9762-1da9cc18bfba) | ![Screenshot_20230620_171538](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/87cd83f9-aef5-4437-865d-e63351a16ba3)|


To test:

- Launch Jetpack app, and login
- Verify that Me/Avatar doesn't appear at top right hand corner anymore 
- Verify that Me is in the bottom navigation, next to Notifications
- Tap on Me tab
- Verify that it goes to Me Screen as before

- Launch WordPress app, and login
- Verify that Me/Avatar doesn't appear at top right hand corner anymore 
- Verify that Me is in the menu, under manage category
- Tap on Me row
- Verify that it goes to Me Screen as before

#### To-do for later in a separate PR

- [ ] Replace icons with new ones on Me Screen
- [ ] Add change photo cta on Me -> My Profile screen if required


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Existing tests

PR submission checklist:


UI Changes testing checklist:
